### ISSUE TITLE
Vector search with Langchain results in UnprocessableEntityError

### ISSUE THREAD

**Issue by ricurious** [AzureCosmosDB/Azure-OpenAI-Python-Developer-Guide#4]
I'm able to perform the vector search using pymongo, however, trying to do it using Langchain as per Lab 04 results in UnprocessableEntityError as shown below:
![image](https://github.com/AzureCosmosDB/Azure-OpenAI-Python-Developer-Guide/assets/86165304/cd6e1ad8-e2b4-4964-ad9f-c653c471de0b)

**Comment by ricurious** [comment]
I'm modifying line 496 to `input=texts[0]` within `.venv/Lib/site-packages/langchain_community/embeddings/openai.py` as a workaround for now.

**Comment by farookfazni** [comment]
Hi, I am having the same issue.

**Comment by Clsmith2** [comment]
Yeah, I've been having the same issues as you have been. Thank you so much for the workarounds they have been very helpful!
